ui/app/templates/permissions_modal.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
         <form role="form" ng-submit="grantRole()">
           <div class="form-group" ng-class="{'has-error': errors.role}">
             <label for="id_role">Role</label>
-            <input type="text" class="form-control" id="id_role" ng-model="role.role"
+            <input type="text" class="form-control" id="id_role" ng-model="role.role" autocomplete="off"
             typeahead="role for role in roles_list | filter:$viewValue | limitTo:16">
             <p class="help-block" ng-show="errors.role">{{ errors.role.join(', ') }}</p>
           </div>
```

Comment:
I think the reason that the Save button is further up is because we have save/delete buttons for existing permissions in the bottom half of the modal. Maybe we should change the close button to red to make it more obvious? To be honest though, I think this UI could do with a bit of an overhaul. There's a lot of small issues with that might be fixed by rethinking it.